manual-install/update-yaml.sh

```diff
@@ -128,12 +130,6 @@ networks:
 NETWORK
 
 cat containers.yml > latest.yml
-sed -i '/image:/s/$/:latest/' latest.yml
-
-cat containers.yml > latest-arm64.yml
-sed -i '/image:/s/$/:latest-arm64/' latest-arm64.yml
-sed -i '/  nextcloud-aio-clamav:/,/^$/d' latest-arm64.yml
-sed -i '/nextcloud[-_]aio[-_]clamav/d' latest-arm64.yml
-sed -i '/CLAMAV_ENABLED/d' latest-arm64.yml
+sed -i '/image:/s/$/:${IMAGE_TAG}/' latest.yml
```

Comment:
```suggestion
sed -i '/image:/s/$/:\${IMAGE_TAG}/' latest.yml
```
You can check if this fixes SC